client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -199,20 +189,29 @@ const productSaleModel = {
 		labelText: translate( 'Product & Sale Price' ),
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
-			salePrice: (
-				<CurrencyField
-					labelText={ translate( 'Product Sale Price' ) }
-					isRequired
-				/>
-			),
+			salePrice: {
+				component: (
+					<CurrencyField
+						labelText={ translate( 'Product Sale Price' ) }
+						isRequired
+					/>
+				),
+				validate: validateSalePrice,
+			},
 		},
 	},
 	conditions: {
-		labelText: translate( 'Conditions', { context: 'noun' } ),
+		labelText: translate( 'Conditions', { comment: 'must be met to use coupon' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Add Condition' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).